t/25-bugs.t

```diff
@@ -40,4 +42,30 @@ ok($bug->refreshed, 'bug refreshed');
 ok($bug->bugid,     'bugid matched');
 is($c->bugtitle_for('poo#200', $bug), "Bug referenced: poo#200\nfoo bar < \" & ß", 'bug title not already escaped');
 
+subtest 'Unreferenced bugs cleanup job works' => sub {
+    $bug = $bugs->get_bug('poo#201');
+    $bug = $bugs->get_bug('poo#202');
+    $schema->resultset('Jobs')->create(
+        {
+            id   => 421,
+            TEST => "textmode",
+        });
+    $schema->resultset('Comments')->create(
+        {
+            job_id  => 421,
+            user_id => 1,
+            text    => 'poo#202',
+        });
+    ok(scalar($bugs->all) > 0, 'Bugs available for cleanup');
+
+    my $minion = $t->app->minion;
+    my $id     = $minion->enqueue('limit_bugs');
+    my $worker = $minion->worker->register;
+    my $job    = $worker->dequeue(0, {id => $id});
+    $job->perform;
+    $worker->unregister;
+
+    is(scalar($bugs->all), 1, 'Bugs cleaned up');
```

Comment:
Why would you write `scalar($bugs->all)` when you can simply write `$bugs->count`? And please add a check to verify whether this is actually the right bug.